packages/atomic/src/themes/default.css

```diff
@@ -26,4 +26,5 @@
   /* Others */
   --atomic-font-family: Lato, Arial, Helvetica, sans-serif;
   --atomic-icon-size: 30px;
+  --atomic-bold: 700;
```

Comment:
Nit pick, would name `--atomic-font-bold`, what is we have a regular later, `--atomic-regular` would be less meaningful if we follow the same pattern